.gitattributes

```diff
@@ -12,6 +12,7 @@
 *.svg binary
 
 # Prefer LF for most file types
+*.css text eol=lf
```

Comment:
why is this here?

Reply:
I added this because a lot of Windows-based editors tend to use CRLF unless you explicitly say to use LF, and CSS files weren't already covered here. I noticed this when I went to edit `src/components/menu-bar/menu-bar.css` as part of the code in this PR. With this line in place, `git` will automatically ensure that LF-only line endings are committed to the repository  in any CSS file, regardless of whether the editor used CRLF, LF, CR, or some mix of them in that file.